schema/fair.js

```diff
@@ -4,23 +4,29 @@ import date from './fields/date';
 import Profile from './profile';
 import Image from './image';
 import Location from './location';
+import { GravityIDFields } from './object_identification';
 import {
   GraphQLObjectType,
+  GraphQLID,
   GraphQLString,
   GraphQLBoolean,
   GraphQLNonNull,
 } from 'graphql';
 
+const OrganizerType = new GraphQLObjectType({
```

Comment:
Could do, but unlike const naming in Ruby that won’t really change much on the outside as it’s not exported. I think that in the scope of the ‘fair’ file it’s clear what the ‘organiser’ organises. Let me know if you feel strong about this, I’m on the fence.